modules/preparing-aws-sts-credentials-for-oadp.adoc

```diff
@@ -0,0 +1,178 @@
+// Module included in the following assemblies:
+//
+// * backup_and_restore/application_backup_and_restore/oadp-aws-sts/oadp-aws-sts.adoc
+
+:_mod-docs-content-type: PROCEDURE
+[id="preparing-aws-sts-credentials-for-oadp_{context}"]
+= Preparing AWS STS credentials for OADP
+
+An {aws-full} account must be prepared and configured to accept an {oadp-first} installation. Prepare the {aws-short} credentials by following the proceeding steps.
+
+.Procedure
+. Define the `cluster_name` environment variable by running the following command:
++
+[source,terminal]
+----
+$ export CLUSTER_NAME= <AWS cluster name> <1>
+----
+<1> The variable can be set to any value.
+
+. Retrieve all the details of the `cluster` such as the `AWS_ACCOUNT_ID, OIDC_ENDPOINT` by running the following command:
++
+[source,terminal]
+----
+$ export CLUSTER_VERSION=$(oc get clusterversion version -o jsonpath='{.status.desired.version}{"\n"}')
+
+export AWS_CLUSTER_ID=$(oc get clusterversion version -o jsonpath='{.spec.clusterID}{"\n"}')
+
+export OIDC_ENDPOINT=$(oc get authentication.config.openshift.io cluster -o jsonpath='{.spec.serviceAccountIssuer}' | sed 's|^https://||')
+
+export REGION=$(oc get infrastructures cluster -o jsonpath='{.status.platformStatus.aws.region}' --allow-missing-template-keys=false || echo us-east-2)
+
+export AWS_ACCOUNT_ID=$(aws sts get-caller-identity --query Account --output text)
+export ROLE_NAME="${CLUSTER_NAME}-openshift-oadp-aws-cloud-credentials"
+----
+. Create a temporary directory to store all the files  by running the following command:
++
+[source,terminal]
+----
+$ export SCRATCH="/tmp/${CLUSTER_NAME}/oadp"
+mkdir -p ${SCRATCH}
+----
+. Display all the gathered details by running the following command:
++
+[source,terminal]
+----
+$ echo "Cluster ID: ${AWS_CLUSTER_ID}, Region: ${REGION}, OIDC Endpoint:
+${OIDC_ENDPOINT}, AWS Account ID: ${AWS_ACCOUNT_ID}"
+----
+. On the {aws-short} account, create an IAM policy to allow access to {aws-short} S3:
+
+.. Check to see if the policy exists by running the following commands:
++
+[source,terminal]
+----
+$ export POLICY_NAME="OadpVer1" <1>
+----
+<1> The variable can be set to any value.
```

Comment:
What variable? This is just a question. Not sure if there's anything to fix

Reply:
OadpVer1